tests/dragonfly/replication_test.py

```diff
@@ -1836,3 +1836,46 @@ async def test_replicaof_reject_on_load(df_local_factory, df_seeder_factory):
     await c_replica.close()
     master.stop()
     replica.stop()
+
+
+@pytest.mark.asyncio
+@pytest.mark.slow
+async def test_eviction_propagation(df_local_factory, df_seeder_factory):
+    master = df_local_factory.create(proactor_threads=1, cache_mode="true", maxmemory="290mb")
+    replica = df_local_factory.create(proactor_threads=1)
+    df_local_factory.start_all([master, replica])
+
+    c_master = master.client()
+    c_replica = replica.client()
+
+    seeder = df_seeder_factory.create(port=master.port, val_size=1000, stop_on_failure=False)
```

Comment:
df_seeder_factory.create does not send any data to server only when you do seeder.run it sends data. I sugest moving this below just above seeder.run

Reply:
ok